tests/test_user.py

```diff
@@ -625,6 +625,25 @@ def test_get_licenses(self, m):
 
         self.assertEqual(2, len(licenses))
 
+    # resolve_path()
```

Comment:
It looks like there may be separate test cases in here. Can we split it into two functions (e..g `test_resolve_path_null_input` and `test_resolve_path`?